lib/Trolley/Http.php

```diff
@@ -78,7 +78,8 @@ private function _throwStatusCodeException($response, $message=null)
 
     public function delete($path, $query = null, $params = null)
     {
-        $response = $this->_doRequest('DELETE', $this->_encodeQuery($path, $query), $params);
+        $response = $this->_doRequest('DELETE', $this->_encodeQuery($path, $query), 
+            (!is_null($params)?json_encode($params):$params));
```

Comment:
thinking this should work?

`$params ? json_encode($params) : $params`

Is there reasons to json_encode all falsy values (null, undefined, '', false) ?